pyahrs.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
```

Comment:
Where did this file come from?

Reply:
I pulled Kris Knigga's pyahrs.py and his hud.py just to look at his code.  I'm still a noob with git, I didn't mean to check those in, now removed from the fork

Reply:
Thanks.

The better way of doing this is to create a branch from my (home) repo, then to create a pull request off of it.